packages/routers/src/index.tsx

```diff
@@ -24,5 +24,5 @@ export type {
   TabNavigationState,
   TabRouterOptions,
 } from './TabRouter';
-export { TabActions, TabRouter } from './TabRouter';
+export { type BackBehavior, TabActions, TabRouter } from './TabRouter';
```

Comment:
This makes it seem like a generic type for all navigators, however this is specific to tab & drawer and we may add different back behavior for other navigators in future.

This should already be accessible with `TabRouterOptions['backBehavior']` which makes it more obvious that it's for tab router. So this export seems unnecessary. Please revert this change.

Reply:
I believe you have misunderstood the issue. The user isn't accessing these types, they are being automatically generated by the TypeScript compiler. I cannot change how the compiler writes these types, hence options like `TabRouterOptions['backBehavior']` are not suitable